Cesil/Writer/AsyncWriterBase.cs

```diff
@@ -187,6 +199,16 @@ static async ValueTask WriteAllAsync_ContinueAfterWriteAsync(AsyncWriterBase<T>
                     await ConfigureCancellableAwait(self, disposeTask, cancel);
                     CheckCancellation(self, cancel);
                 }
+
+                return self.RowNumber;
+            }
+
+            // wait for the given task to finish, then return a row count
+            static async ValueTask<int> WriteAllAsync_ContinueAfterCleanupAsync(AsyncWriterBase<T> self, ValueTask waitFor)
+            {
+                await waitFor;
```

Comment:
Isn't this one missing your ConfigureCancellableAwait?

Reply:
Indeed it is, good catch.